### PR TITLE
Set Up Refactor Tractor Project

### LINKED ISSUE #7: PR Template
- [ ] set up PR template for fute changes

### PR DESCRIPTION
**Pull request by nathanielmillard**
###### What does this PR do?  
- Set's up our project to begin refactoring.
- Add a PR template.
- Updates outdated versions of packages used.

###### Where should the reviewer start?  
- Currently the whole project is worth reviewing by those looking to contribute.
- But there isn't much new by this code team.

###### How should this be manually tested?  
- run npm install
- run npm test

###### Any background context you want to provide?  
- Not any at this time.

###### What are the relevant tickets?  
- closed #7 

###### Questions:  
- I am still unsure where as a team we would like to start?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
     "mocha": "^5.2.0",
     "mochapack": "^1.0.0",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.14.1",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.1",
     "webpack": "^4.31.0",
-    "webpack-cli": "^3.2.1",
-    "webpack-dev-server": "^3.1.14"
+    "webpack-cli": "^3.3.12",
```
Comment (timkeresey) [inline package.json:31]: Is the installation of webpack complete? the video made it seem a bit more complex

**Review by stacyp2006 — CHANGES_REQUESTED** [review]
May need to complete webpack tasks.

File: package.json (hunk 1 above)
Comment (nathanielmillard) [reply package.json:31]: Great question! And thanks for updating me on that video and more steps to take. 
Testing it now, it seems to work how we expect!

**Review by stacyp2006 — APPROVED** [review]
Webpack added. I think we are ready to start!

**Pull request merged**